src/controller/CHIPDeviceController.h

```diff
@@ -84,6 +84,10 @@ using namespace chip::Protocols::UserDirectedCommissioning;
 
 inline constexpr uint16_t kNumMaxActiveDevices = CHIP_CONFIG_CONTROLLER_MAX_ACTIVE_DEVICES;
 
+inline constexpr size_t kMaxUdcTransportTcpActiveConnectionCount = CHIP_CONFIG_CONTROLLER_MAX_ACTIVE_TCP_CONNECTIONS;
```

Comment:
There shouldn't be any UDC over TCP.  UDC consists of messages that don't even use MRP because they absolutely do not want acks.